Makefile.prow

```diff
@@ -88,3 +99,45 @@ $(GOBIN)/golangci-lint:
 $(GOBIN)/setup-envtest:
 	@echo "Installing envtest tools"
 	go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+
+.PHONY: clone-oadp-operator
+clone-oadp-operator:
+	@echo "Cloning oadp-operator"
+	rm -rf $(OADP_E2E_DIR)
+	git clone --depth 1 --single-branch --branch $(OADP_E2E_BRANCH) https://github.com/openshift/oadp-operator.git $(OADP_E2E_DIR)
+
+.PHONY: clean-oadp-operator
+clean-oadp-operator:
```

Comment:
add this command as pre run of  `clone-oadp-operator` by writting
```mk
clone-oadp-operator: clean-oadp-operator
```
and remove rm command in `clone-oadp-operator`

Reply:
good suggestion, thanks